org/xbill/DNS/DNSSEC.java

```diff
@@ -869,12 +913,10 @@ private Algorithm() {}
 								   GOST);
 				break;
 			case Algorithm.ECDSAP256SHA256:
-				signature = ECDSASignaturefromDNS(signature,
-								  ECDSA_P256);
+				signature = convertECDSASignature(signature);
 				break;
 			case Algorithm.ECDSAP384SHA384:
-				signature = ECDSASignaturefromDNS(signature,
-								  ECDSA_P384);
+				signature = convertECDSASignature(signature);
```

Comment:
Duplicate branches in switch.  If both this case and the one above it are to call the same exact method, would it would be cleaner to:
```
  case Algorithm.ECDSAP256SHA256:
  case Algorithm.ECDSAP384SHA384:
    signature = convertECDSASignature(signature);
    break;
```